src/compass/utils/h5_helpers.py

```diff
@@ -781,7 +781,7 @@ def corrections_to_h5group(parent_group, burst, cfg, rg_lut, az_lut,
              'spacing of slant range of LUT data', {'units': 'meters'}),
         Meta('zero_doppler_time', azimuth, 'azimuth time of LUT data',
              {'units': 'seconds'}),
-        Meta('zero_doppler_time_spacing', np.abs(rg_lut.y_spacing),
```

Comment:
was this removed before we don't mind it being negative?

Reply:
That was the request from @vbrancat to revise the code to have `y_spacing` be negative.